docs/interactions/Slash_Commands.md

```diff
@@ -842,24 +842,29 @@ All options have names, and an option can either be a parameter and input value-
 
 ## Interaction Response
 
-After receiving an interaction, you must respond to acknowledge it. This may be a `pong` for a `ping`, a message, or simply an acknowledgement that you have received it and will handle the command async.
+After receiving an interaction, you must respond to acknowledge it. You can choose to respond with a message immediately using type `4`, or you can choose to send a deferred response with type `5`. If choosing a deferred response, the user will see a loading state for the interaction, and you'll have up to 15 minutes to edit the original deferred response using [Edit Original Interaction Response](#DOCS_INTERACTIONS_SLASH_COMMANDS/edit-original-interaction-response).
 
-Interaction responses may choose to "eat" the user's command input if you do not wish to have their slash command show up as message in chat. This may be helpful for slash commands, or commands whose responses are asynchronous or ephemeral messages.
+![A deferred response tells the user "Bot name is thinking"](ephemeral-example.png)
```

Comment:
This should probably be moved to a feature request on my end, but I'm not sure all bot developers want to anthropomorphize their bot.

Reply:
I'm open to other wording 👍